### PR TITLE
Prepare release 3.0.0-rc2

### PR DESCRIPTION
**Pull request by no-reply**
At long last. 

WIP because i'm working on updating the release notes

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by cjcolvar** [comment]
@no-reply Is this still WIP or are you ready to cut an RC2?

**Comment by rotated8** [comment]
Bumping for @no-reply

**Comment by cjcolvar** [comment]
Bumping again.

**Comment by jeremyf** [comment]
I'm working on the release notes and getting rc2 ready.

File: README.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ NOTE: The steps need to be done in order to create a new Hyrax based app.
 Generate a new Rails application using the template.
 
 ```
-rails _5.2.3_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0.pre.rc1/template.rb
+rails _5.2.3_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0.pre.rc2/template.rb
```
Comment (cjcolvar) [inline README.md:158]:
```suggestion
rails _5.2.3_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0-rc2/template.rb
```

**Comment by no-reply** [comment]
i think this might be ready to go, but needs a review.

**Pull request merged**